examples/website/3d-heatmap/app.js

```diff
@@ -128,15 +83,19 @@ export class App extends Component {
         coverage,
         data,
         elevationRange: [0, 3000],
-        elevationScale: this.state.elevationScale,
+        elevationScale: data && data.length ? 50 : 0,
```

Comment:
One almost wants a way to specify that "transitions" should start at data load time, so this could be dropped...

Reply:
In the website data is passed in from the parent, so `onDataLoad` will not fire.